sim_interfaces/airsim_zenoh_bridge/mock_bridge.py

```
@@ -1,13 +1,9 @@
-#!/usr/bin/env python3
+﻿#!/usr/bin/env python3
```

Ruff (0.15.9)

[warning] Shebang should be at the beginning of the file

(EXE005)